sdks/java/core/src/main/java/org/apache/beam/sdk/options/SdkHarnessOptions.java

```diff
@@ -168,7 +168,7 @@ class DefaultMaxCacheMemoryUsageMbFactory implements DefaultValueFactory<@NonNeg
     public @NonNegative Integer create(PipelineOptions options) {
       SdkHarnessOptions sdkHarnessOptions = options.as(SdkHarnessOptions.class);
       return (Integer)
-          checkNotNull(
+          checkArgumentNotNull(
```

Comment:
This one is strange. Probably `checkStateNotNull` since it indicates an invalid state reached due to this code failing.